contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3809,13 +3789,9 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 							{
 								RoleSpec	   *rol_spec = (RoleSpec *) lfirst(lc);
 								bool has_schema_perms = false;
-								if ((rol_spec->rolename != NULL) && (strcmp(rol_spec->rolename, "public") != 0))
-								{
-									if (check_bbf_schema_for_entry(logicalschema, "ALL", "execute", rol_spec->rolename) && !has_schema_perms)
-										has_schema_perms = true;
-									if (check_bbf_schema_for_entry(logicalschema, funcname, "execute", rol_spec->rolename))
-										del_from_bbf_schema(logicalschema, funcname, "execute", rol_spec->rolename);
-								}
+								if (check_bbf_schema_for_entry(logicalschema, "ALL", "execute", rol_spec->rolename) && !has_schema_perms)
```

Comment:
In one of my previous comments I requested for adding a Macro for "ALL", it improves readability of code and also avoids declaring/modifying constants at multiple places

Reply:
Sorry I missed that.
Addressed now.